bitcoin/exchange/bitcoinprices.go

```diff
@@ -89,11 +96,12 @@ func (b *BitcoinPriceFetcher) fetchCurrentRates() error {
 }
 
 type BitcoinAverage struct {
-	cache map[string]float64
+	cache  map[string]float64
+	client *http.Client
```

Comment:
I can't look now, but I remember refactoring all this to make it easier to test.